cmake/lint.cmake

```diff
@@ -31,15 +31,26 @@ set(AUTOGRAPH_LINT_PATHS
         ${PROJECT_SOURCE_DIR}/dali/test/python/autograph/
 )
 
+add_custom_target(lint-python-black
+        COMMAND
+          black --check --config ${PROJECT_SOURCE_DIR}/pyproject.toml ${PYTHON_LINT_PATHS} ${AUTOGRAPH_LINT_PATHS}
+        COMMENT
+          "Performing black Python formatting check"
+)
 
-add_custom_target(lint-python
+
+add_custom_target(lint-python-flake
         COMMAND
           flake8 --config=${PROJECT_SOURCE_DIR}/.flake8 ${PYTHON_LINT_PATHS}
         COMMAND
           flake8 --config=${PROJECT_SOURCE_DIR}/.flake8.ag ${AUTOGRAPH_LINT_PATHS}
```

Comment:
Since we are unifying the autograph, shouldn't this be removed?

Reply:
There are 3 warnings that are very common in the ag implementation and tests:
```
# E741 - ambiguous variable name 'l'
# E731 - do not assign a lambda expression, use a def
# F401 - 'autograph.operators....' imported but unused - used to bring types into another module
```
I didn't want to rework couple hundred lines and wanted to keep it as is. When I tried to add per-file-excludes, I didn't manage to write a pattern that would detect `autograph` generically in the path name. I would need to list all absolute paths by hand, and this changes based on the build location. The relative paths don't work well. 
For some reason only one exclude out of several worked. 
At this point I just kept the separate config, and run it with few additional excludes.
If you have better idea on how to merge those configs I would be happy to see it.